### PR TITLE
feat(observations): add UiRas water temperature source

### PR DESCRIPTION
**Pull request by mhieta**
- Add uiras_swimming_water_temperature ObservableProperty (migration 0012)
- Add import_uiras_swimming_water_temperatures management command: fetches FeatureCollection from UiRas API, maps features to units via settings.UIRAS_OBSERVABLE_UNITS, stores MeasuredObservation records idempotently, updates UnitLatestObservation
- Extract shared parse_measurement/store_observation helpers to observations/swimming_temperatures.py; refactor sensoripaja command to use shared helpers
- Add UIRAS_BASE_URL and UIRAS_OBSERVABLE_UNITS settings
- Add tests: import, idempotency, dry-run, missing units/feature, missing temp_water; add uiras_property fixture
- Add import_uiras_swimming_water_temperatures to update.sh

Refs: HUL-78

## Summary by CodeRabbit

* **New Features**
  * Import UiRas swimming-water temperature readings with optional dry-run.
  * Added observable property for UiRas swimming-water temperature.

* **Refactor**
  * Extracted measurement parsing and idempotent storage into reusable helpers.

* **Migrations**
  * Migration to create the UiRas swimming-water temperature property and service associations.

* **Chores**
  * System update now triggers UiRas import; added UiRas base URL and unit mapping.

* **Tests**
  * Tests for import behavior, idempotency, dry-run, and edge cases.

### CODE REVIEW

File: observations/management/commands/import_uiras_swimming_water_temperatures.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        for unit_id in sorted(known_unit_ids):
+            feature = features_by_id.get(str(unit_feature_map[unit_id]))
+            was_created = self._process_unit(
+                unit_id, feature, observable_property, dry_run, errors
+            )
+            if was_created is None:
+                continue
+            if was_created:
+                created += 1
+            units_updated += 1
```
[the remark this answers is not in the document]
Comment (mhieta) [reply observations/management/commands/import_uiras_swimming_water_temperatures.py:147]: Not a bug — this is correct behaviour. The units_updated counter tracks actual database writes, not fetches. In --dry-run mode nothing is written, so reporting "0 unit(s) updated" is accurate and intentional — the same design exists in the sensoripaja command. The per-unit [dry-run] unit=… temp=… lines already tell the operator exactly what would be written, and the final summary correctly reflects the actual state of the database (unchanged). Changing the counter semantics in dry-run mode would make the non-dry-run counter mean different things in different modes, which would be more confusing, not less.

No change needed.

**Review by jorilindell — APPROVED** [review]
